.github/actions/enforce-pr-conventions/action.yml

```diff
@@ -22,13 +22,13 @@ runs:
     - name: Get branch name
       uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v5.22.3
 
-    - name: Check is Dependabot PR or Propagation PR or Upstream PR
+    - name: Check is Dependabot PR or Propagation PR or Upstream PR or Release PR
       id: check
       env:
         PR_USER: ${{ github.event.pull_request.user.login }}
       shell: bash
       run: |
-          if [[ $PR_USER != 'dependabot[bot]' && $BRANCH_NAME != pr-* && $BRANCH_NAME != upstream-* ]]; then
+          if [[ $PR_USER != 'dependabot[bot]' && $BRANCH_NAME != pr-* && $BRANCH_NAME != upstream-* && $BRANCH_NAME != release-* ]]; then
```

Comment:
there is a `whitelist-branches` input for avoiding hardcoding branches here, which can change depending on the target repo

Reply:
as @atchertchian mentioned above most probably we need a wildcard as suffix
```  readarray -t branches <<< "${{ inputs.whitelist-branches }}"
        for branch in "${branches[@]}"; do
            if [[ $branch == $BRANCH_NAME ]]; then
            
  Or I am not sure if u can pass something like release-* as value